sktime/transformations/_delegate.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+"""Delegator mixin that delegates all methods to wrapped forecaster.
+
+Useful for building estimators where all but one or a few methods are delegated.
+For that purpose, inherit from this estimator and then override only the methods
+    that are not delegated.
+"""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["fkiraly", "miraep8"]
```

Comment:
does not need me here, thanks for the thought

Reply:
sure thing!  I figured since it was so heavily based on your _DelegatedForecaster I would try to err on the side of caution, but you have now been removed.